README.md

```diff
@@ -55,7 +55,7 @@ sudo yum install -y epel-release && sudo yum install -y git gcc gcc-c++ make cma
 sudo apt update
 sudo apt-get install gcc g++ make cmake autoconf automake libtool
 
-# MACOSX
+# MacOS
```

Comment:
Why not `macOS`

Reply:
The standard name is `macOS`😂

Reply:
It will be good to use `macOS` since apple/wikipedia also using this name and find nowhere using MacOS.